stata_kernel/config.py

```diff
@@ -24,6 +25,17 @@ def __init__(self):
         if not self.get('stata_path'):
             self.raise_config_error('stata_path')
 
+        # Local settings via %set will override ~/.stata_kernel.conf
+        # where applicable.
+        self.overrides = {
+            'plot': {
+                'format': None,
+                'scale': None,
+                'width': None,
+                'height': None
+            }
+        }
+
```

Comment:
Can you just make this part of `Config.env`? Specifically, the `config.get()` function has a `permanent` option, and if that's `False`, it won't be written to the configuration. So in my mind, the `%set` magic would be a wrapper around `config.set(key, val)`. Just use `plot_format`, `plot_scale`, `plot_width`, and `plot_height`. Also, the current format configuration variable is `graph_format`, not `plot_format`. So that will get confusing. I'm happy to change that to `plot_format`, but there are several places in the code where we need to make that change.

Reply:
Ok.